app.rb

```diff
@@ -20,6 +20,11 @@ def authenticate
       authenticate
     end
 
+    # service heartbeat
+    get '/status' do
+      { message: 'still alive ...' }
```

Comment:
@josisusan, it is generally handle by rack middleware or third party services like incapsula/akamai. Third party service is always better as it has a large data set to make decision like this.